src/components/EBIFooter/index.tsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
+import Link from 'components/generic/Link';
+import cssBinder from 'styles/cssBinder';
+import local from './styles.css';
+const css = cssBinder(local);
+
+const EBIFooter = () => {
+  const year: string = new Date().getFullYear().toString();
+  return (
+    <div className="vf-footer">
```

Comment:
I'm adding in the CSS a bit of padding left and right so it doesn't get on the edge of the window when the window is small.